src/lib.rs

```diff
@@ -33,8 +33,14 @@ extern crate cfg_if;
 use core::mem::MaybeUninit;
 
 mod backends;
+#[allow(dead_code, reason = "not used in all backends")]
```

Comment:
Remove this, we only want `#[allow(dead_code)]` on certain methods in the `error` module.

Reply:
There are only two `pub (crate)` items on the error module and prior to this change they are both marked `allow(dead_code)`. Which methods do you *not* want this on?